modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -3594,4 +3595,15 @@ public boolean uncommitedTx(final DataEntry dataEntry) {
     public void clearUncommitedStates() {
         uncommitedTx = Collections.emptySet();
     }
+
+    /**
+     * Checks if transaction mode supported for transaction aware queries.
+     * @param isolation Transaction isolation to check.
+     */
+    public static void ensureTransactionModeSupported(TransactionIsolation isolation) {
```

Comment:
Only one usage. Do we really need this method? Can we keep it inlined?

Reply:
The second usage appear in #11580